specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json

```diff
@@ -2071,6 +2071,18 @@
           "$ref": "#/definitions/RedisCreateProperties",
           "description": "Redis cache properties."
         },
+        "redisVersion": {
```

Comment:
Any property in the PUT body should also show up in the response body.

Reply:
Response body already contains "redisVersion" property. Please check "redisVersion" in "RedisProperties".

Reply:
We already had the redisVersion property in the response, formerly it was readOnly, this change is to make it a writable property, and add special behavior when the client requests version '6', which we are adding support for.

We think it should be non-breaking. Previously responses and examples would have contained a value such as '2.x', '3.x' or '4.x' so anybody passing us a value which they got from old ARM templates or samples will continue to get the old behavior.

Reply:
This spec says it only supports "4" and "6".  It needs to support all possible values that can show up in the response.

Reply:
That could be a bit tricky. We currently return longer more detailed values than '4', '6', in the response. like '4.0.23' (for example) to give users more idea about what exact version of redis they are running. We do also accept such values, in requests, since we parse them into a version number. But then we only really use that to see what major version they want, so they might not get the same version back in response.

Reply:
@pilor How would it be if we just keep modelling this as string, and document the behavior?